scrapy/settings/default_settings.py

```diff
@@ -246,7 +246,7 @@
 RETRY_PRIORITY_ADJUST = -1
 
 ROBOTSTXT_OBEY = False
-ROBOTSTXT_PARSER = 'scrapy.robotstxt.PythonRobotParser'
+ROBOTSTXT_PARSER = 'scrapy.robotstxt.ProtegoRobotParser'
```

Comment:
I think docs needs to be changed as well, see https://github.com/scrapy/scrapy/blob/master/docs/topics/settings.rst#robotstxt_parser.